### PR TITLE
travis ci

### PR DESCRIPTION
**Pull request by nmn**
Just adding some continuous integration.

### CODE REVIEW

**Comment by steelbrain** [comment]
How about you also limit it to master branch only?

**Comment by Arcanemagus** [comment]
`brew` should already be installed in the [OS X environment](http://docs.travis-ci.com/user/osx-ci-environment/) btw, no need to re-install it.

**Comment by Arcanemagus** [comment]
Note that when you get around to AppVeyor setup, `apm test` is currently broken on Windows and always reports failure.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: objective-c
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+env:
+  - APM_TEST_PACKAGES="linter"
```
Comment (Arcanemagus) [inline .travis.yml]: You should probably remove this line, `linter-flow` is supposed to automatically install this for the user so you might as well test that too :wink:.

Comment (nmn) [inline .travis.yml]: I'll do that another time. I'm just happy to get ci working at all for now!

**Comment by Arcanemagus** [comment]
If you want an example I've spent quite a bit of time getting [`linter-php` ](https://github.com/AtomLinter/linter-php) set up with Travis-CI (Linux + OSX), CircleCI (Linux), and AppVeyor (Windows).

I actually just got OS X testing working properly last night in that setup, and found a bug that only shows up there, so part of the setup is still in https://github.com/AtomLinter/linter-php/pull/90.

**Pull request merged**